source/extensions/filters/network/zookeeper_proxy/filter.cc

```diff
@@ -314,17 +356,39 @@ void ZooKeeperFilter::onResponse(const OpCodes opcode, const int32_t xid, const
     opcode_info.counter_->inc();
     opname = opcode_info.opname_;
     opcode_latency = opcode_info.latency_name_;
+
+    if (!config_->latency_threshold_map_.empty()) {
+      // Set default latency threshold.
+      uint32_t default_latency_threshold = 100;
+      auto it = config_->latency_threshold_map_.find(-1);
+      if (it != config_->latency_threshold_map_.end()) {
+        default_latency_threshold = it->second;
+      }
```

Comment:
let's make `default_latency_threshold` a data member, and only calculate once when you init the filter config